Editing in a few minor changes in the documents (#46)
tobyhodges/lesson-sprint-recommendations: before.md

  Document your discussions and plans in a central location so that everyone involved in the process can access them.
  Divide responsibilities to put together resources (such as for communication and norms) that will be shared with all participants. (See below for more details.)
  
- **Establish communication channels and norms.** This will set an expectation for how collaborators will communicate before, during, and after the sprint.
    - Dedicated messaging channels have been used with success for conversations/short updates during the sprint and in-between sprint conversations.
    See [During the Sprint](/during.md) for more details.
    - Mailing lists for initial organisation and other important and more permanent communication, e.g. scheduling meetings.
    - **[Online]** Video conferencing technology that is an accessible choice for everyone and suitable for the format of the sprint.
    - Decide on the basic etiquette for participation, such as how to collect ideas, who to contact for support, how to participate in an ongoing discussion, how to contribute to content development and how to review contributions. 

>     - Collaborative note-taking platform is suitable for recording and communicating decisions, minutes and actions from meetings.

Reviewer: ```suggestion
    - Choose a collaborative note-taking platform for recording and communicating decisions, minutes and actions from meetings. (See [the tools section](tools.md) for details of some different note taking platforms.)
```

- **Think about the timing of your sprint.**
    - You can run the sprint in conjunction with an upcoming event, e.g. a conference/meeting that many in your community will already be engaged with.
    If you would like to attract new contributors, this will make it easier to get people's attention and
    convince them to block time in their calendar to join in with the sprint.
    You may also be able to tap into the promotional communications etc. for that event,
    and/or invite people to join the sprint, e.g. via a lightning talk or poster.
    - Conversely, you may want to avoid clashes with another event if your team is going to be engaged elsewhere or have other commitments.
    - Announce the dates for your sprint at least 6 weeks in advance. 

> - **Add dates and time in your calendar**: In addition to the event dates, schedule a few planning meetings with the committee members _before_ the sprint and one meeting _after_ the sprint for wrap-up/follow-up tasks.

Reviewer: ```suggestion
- **Add dates and time in your calendar**: In addition to the event dates, schedule a few planning meetings with the committee members _before_ the sprint and one meeting/co-working session _after_ the sprint for wrap-up/follow-up tasks.
```

## Recommendations for GitHub

- **Review open issues on your lesson repository** before the sprint
  and consider flagging (labelling) existing issues and opening new ones to describe tasks
  that could be tackled during the event.
  [GitHub Projects](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-project-boards)
  can help with this,
  allowing you to label particular issues within scope for the sprint
  and even organise them further into categories to help contributors find something to work on.
  Even if you cannot invest time into managing issues to this extent,
  make sure open issues include all the contextual information necessary

>   for a new or part-time contributor to understanding what needs doing

Reviewer: ```suggestion
  for a new or part-time contributor to understand what needs doing
```
